code/GameEngine/Components/Effects/ParticleSystem.cs

```diff
@@ -26,9 +45,9 @@ public class ParticleSystem : BaseComponent, BaseComponent.ExecuteInEditor
 		}
 	}
 
+	[Property] public GameObject ControlPoint0 { get; set; }
```

Comment:
We can't do this, it'll break everything that currently uses it

Reply:
I agree it's not ideal but if there was ever a time to break it it'd be now. This feels like something that'd be potentially confusing for newer users later down the line if it isn't changed.